### DISCUSSION TITLE
Weekly update 5/25/2022

### DISCUSSION THREAD

**Discussion by kirubeltadesse** [Swarmies/sap-core#24] [General]
Please use this template for updating your progress!

**Comment by kirubeltadesse** [comment]
# Kirubel

###  How do you feel today?

I am feeling 💪🏼 great today. I am looking forward to everyone's PR for #26 

### What did you do since the last meeting? 

- I updated the project container image #23 
- Added `.vscode` to save the formating for the project. 

### What are you planning to do moving forward? 

- I plan to add documentation on how to start the project locally.
- preparing information for the team to understand the workflow for MVC. 
- preparing information for debugging tool for individual containers.

### Anything hampering your process? 

- There probot ( #9 )  setup is taking longer than expected. I will get back to it at a later time.

**Reply by Getnet12** [reply]
# Getnet

###  How do you feel today?

I am feeling alright.

### What did you do since the last meeting? 

- I haven't done a thing since our previous meeting

### What are you planning to do moving forward? 

- I'll be working on the ERD modeling

### Anything hampering your process? 

- I was focused on other things. I wasn't even using my computer much. I'll get back to the ticket starting next week.

**Reply by kirubeltadesse** [reply]
I will suggest that you install slack on your phone and keep up with the conversion and code changes until next week.

**Comment by dinaolmelak** [comment]
# Dinaol

###  How do you feel today?

I am feeling good today.

### What did you do since the last meeting? 

- I made a PR for the #18  documentation.

### What are you planning to do moving forward? 

- I plan to gather more information on CircleCI and set it up

### Anything hampering your process? 

Not, in particular, I was working on another ticket before CircleCI.
